scipy/stats/__init__.py

```diff
@@ -344,15 +434,13 @@
 
    stats.qmc
 
-Resampling Methods
+Contingency Tables
 ==================
 
-.. autosummary::
-   :toctree: generated/
+.. toctree::
+   :maxdepth: 4
```

Comment:
The intent was for this to render like QMC:

![image](https://user-images.githubusercontent.com/6570539/211946711-35e8fc6c-971f-4348-9d20-b356e9df5315.png)

But it's like:
![image](https://user-images.githubusercontent.com/6570539/211946742-ecbde284-6ba7-4fbe-a5cf-9a8c6e562c17.png)

(and so is mstats). Would changing `:maxdepth:` fix this?

Reply:
So we could do that if we add Boschloo and Barnard to this namespace as well.

It's not really by playing with this param, but more by having a proper init in the contingency submodule. This is what we did for QMC. You find the following in its init.

```
Engines
-------

.. autosummary::
   :toctree: generated/

   QMCEngine
   Sobol
   Halton
   LatinHypercube
   PoissonDisk
   MultinomialQMC
   MultivariateNormalQMC

Helpers
-------

.. autosummary::
   :toctree: generated/

   discrepancy
   update_discrepancy
   scale
```